src/io/flutter/run/daemon/FlutterDaemonController.java

```diff
@@ -205,7 +206,7 @@ private static GeneralCommandLine createCommandLine(Project project) throws Exec
       }
 
       flutterSdkPath = flutterSdk.getHomePath();
-      workingDir = project.getBasePath();
+      workingDir = ModuleUtil.getProjectBasePath(project);
```

Comment:
Flutter daemon is not project-specific, is it? Looks like it doesn't need any special working dir. 
Don't miss a chance to completely get rid of `ModuleUtil.getProjectBasePath` method. :) 

The presence of this method may provoke others to use it, whereas methods like this are kind of 'harmful'. It is impossible to rationally explain why it takes first content root of first Flutter module. And it's name doesn't reflect its behavior.

If I ever have to write a harmful method, I prefer to name it as such, for example `mayBeGetSomeContentRootOfSomeFlutterModule()`. Do you agree it is much better than `getProjectBasePath()`? Because it won't be used by other developers occasionally! :) And because there's no such thing as project base path.

Another example of 'harmful' method is `isFluttering()`. It was unused when it was introduced, but developers noticed and now it has several usages. Well, may be `isFluttering()` is not that bad. Especially taking into consideration that its implementation uses cached results. But every call to `ProjectManager.getOpenProjects()` and 'ModuleManager.getModules()` rings my bells.

But methods called like `ModuleUtil.getProjectBasePath()` worry me much more.